proposals/bot-orchestrators.md

```diff
@@ -0,0 +1,182 @@
+Projects do not include regular day-to-day maintenance and improvement work, e.g. on testing, tooling, validation, code clarity, refactors for future capability, etc.
+-->
+<!--
+For ease of discussion in PRs, consider breaking lines after every sentence or long phrase.
+-->
+
+## Purpose &amp; impact 
+#### Background &amp; intent
+_Describe the desired state of the world after this project? Why does that matter?_
+<!--
+Outline the status quo, including any relevant context on the problem you’re seeing that this project should solve. Wherever possible, include pains or problems that you’ve seen users experience to help motivate why solving this problem works towards top-line objectives. 
+-->
+
+Let's start with the state of the world *before* this project is completed.
+
+Currently, we are about to start a project to build [storage and retrieval dealbots](https://github.com/protocol/web3-dev-team/pull/84). These bots will attempt storage and retrieval deals on the Filecoin mainnet based on input provided to them on stdin (or a command line supplied input file).  They will output their results to stdout, with that output then piped to a log aggregator for visualization (eg, Kibana, Observable, etc).
+
+For most use cases of these bots, we don't just want to invoke them manually on the command line. Instead, we need a higher-level orchestration system that will carry out a series of storage and retrieval attempts to generate the data we want for that use case.
+
+One such orchestrator will be the one for deal success testing. This PR covers only that orchestrator.  Future orchestrators (in future PRs) will include those for reputation systems, scraping data from other sources and storing it in Filecoin, etc.
+
+The desired state of the world after building this deal success orchestrator is:
+
+ - We will have a long-running daemon program that repeatedly attempts storage and retrieval deals on mainnet
+ - This collection of results from many storage and retrieval deals will get aggregated into the log aggregator that the bots output to.
+ - These aggregate results will be visualized via the [deal success dashboards project](https://github.com/protocol/web3-dev-team/pull/85).
+
+
+#### Assumptions &amp; hypotheses
+_What must be true for this project to matter?_
+<!--(bullet list)-->
+
+ - The storage and retrieval bots are built and able to output test results to a dashboard.
+ - The bots are capable of accepting instructions on stdin or a unix socket or some other kind of command issuance interface.
+
+#### User workflow example
+_How would a developer or user use this new capability?_
+<!--(short paragraph)-->
+
+```
+$ ./dsr-bot-orchestrator --input my-tests.json start
+```
+
+The input file `my-tests.json` might look something like this:
+
+```
+[
+	{
+		"dealType":"storeThenRetrieve",
+		"storeThenRetrieveParameters":
+			{
+				"retrievalDelayHours":"72",  // store the file, wait 72 hrs, then try to retrieve it
+			},
+		"miner":"f0xxxx",
```

Comment:
likely we will have a list of miners we want to make deals with, rather than a policy per-miner?